src/views/router/routes.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Chat from '../chat/chat'
+import { Home } from '../home'
+import Settings from '../settings/settings'
+import { FaCog, FaComment, FaHome } from 'react-icons/fa'
+
+/* eslint-disable sort-keys */
+export const appRoutes = {
```

Comment:
This object lost it's Type safety. Re-apply the type to this.
```suggestion
export const appRoutes: AppRoutes = {
```